src/assets/i18n/es.json

```diff
@@ -0,0 +1,245 @@
+      "TRANSACTION": "transacción"
+    },
+    "TYPE": "Tipo",
+    "WALLETS_PAGE": {
+      "ADD_WALLET_ERROR": "No se pudo añadir la cartera",
+      "ADD_WALLET_TEXT": "Para importar o crear una cartera haga clic en el botón de abajo",
+      "ADDRESS": "Dirección",
+      "ALERT_SUCCESSFULLY_ENCRYPTED_SECOND_PASSPHRASE": "La frase secreta secundaria fue encriptada con éxito",
+      "BACKUP_ACCOUNT": "IMPORTANTE! Haga una copia de seguridad de esta frase secreta de forma segura, sin estas palabras usted no puede restaurar su cartera o enviar transacciones y TODOS LOS TOKENS ARK SERÁ PERDIDOS",
+      "BACKUP_PASSPHRASE": "Haga una copia de seguridad de la frase secreta de forma segura, sin estas palabras usted no podrá restaurar su cartera.",
+      "BACKUP_SECOND_PASSPHRASE": "Haga una copia de seguridad de la frase secreta secundaria de forma segura, sin estas palabras usted no podrá mover sus fondos.",
+      "BACKUP_SECOND_PASSPHRASE_SECURELY_RETYPE_TEXT": "Usted puede editar para una frase secreta de su elección. Recuerde hacer una copia de seguridad con seguridad, tendrá que volver a escribir para la validación.",
+      "BALANCE": "Saldo",
+      "BIP38_SEED": "BIP38 Seed",
+      "CREATE_WALLET": "Crear una cartera",
+      "ENTER_SECRET_PASSPHRASE": "Entre con su frase secreta",
+      "ENTROPY": "Entropía",
+      "ENTROPY_RANDOM_BYTES": "Deslice para generar bytes aleatorios",
+      "GENERATE_ENTROPY": "Gerar Entropia",
+      "IMPORT_INVALID_ADDRESS": "Esta dirección no se puede importar a este perfil",
+      "IMPORT_WALLET": "Importar cartera",
+      "KEEP_SAFE": "Mantener a salvo",
+      "LABEL": "Rótulo",
+      "MY_WALLET": "Mi cartera",
+      "NEW_WALLET_NAME": "Nuevo nombre de la cartera",
+      "NO_TRANSACTIONS": "Ninguna transacción disponible",
+      "PASSPHRASE": "Frase Secreta",
+      "PASSPHRASE_NOT_MATCH": "La frase secreta insertada no corresponde a la que acaba de obtener.",
+      "PUBLIC_ADDRESS": "Dirección pública",
+      "PUBLIC_ADDRESS_SHARE": "Dirección pública (COMPARTIR)",
+      "PUBLIC_KEY": "Clave pública",
+      "RE_ENTER_PASSPHRASE": "Introduzca su frase secreta de nuevo",
+      "REMOVE_WALLET": "Quitar la cartera",
+      "REMOVE_WALLET_TEXT": "Su cuenta será eliminada. Asegúrese de haber hecho copia de seguridad, usted necesitará su frase secreta original para restaurar.",
+      "SCAN_QRCODE": "Escanear Código QR",
+      "SCAN_QRCODE_TEXT": "¡Importe su dirección o frase secreta usando Código QR, apenas apunte y escanear!",
+      "SECOND_BIP38_SEED": "Segundo BIP38 Seed",
+      "SECOND_PASSPHRASE": "Frase Secreta Secundaria",
+      "SECOND_WIF": "Segundo WIF",
+      "SECRET_PASSPHRASE": "Frase Secreta",
+      "SECRET_PASSPHRASE_KEEP_SAFE": "Frase Secreta (Salve)",
+      "SEED": "Seed",
+      "TOTAL_BALANCE": "Slado total",
+      "USERNAME": "Nombre de usuario",
+      "WALLETS": "Carteira",
+      "WIF": "WIF"
+    },
+    "WATCH_ONLY": "Camino Watch Only",
```

Comment:
"Sólo lectura". I'm not completely sure. Suggestions?

Reply:
fixed 77